src/components/plots/plot.ts

```diff
@@ -54,6 +56,8 @@ module Plottable {
       this.clipPathEnabled = true;
       this.classed("plot", true);
       this._key2PlotDatasetKey = d3.map();
+      this._attrToExtents = d3.map();
+      this._extentProvider = (scale: Scale<any, any>) => this._extentsForScale(scale);
```

Comment:
if I pass `this._extentsForScale` directly, it will be invoked with `this` equal to whatever it is inside the `CallbackSet` inside the `Scale`. Since `_extentsForScale()` depends on `this` being the `Plot`, that won't work.